talk/expert/cpp20concepts.tex

```diff
@@ -0,0 +1,194 @@
+\subsection[tmpl]{C++20 Requirements and Concepts}
+
+\begin{frame}[fragile]
+  \frametitlecpp[20]{C++20 Requirements and Concepts}
+  \begin{block}{Motivation}
+    Generic programming is made of variable, function and class templates
+    which can be instantiated with different types. It is frequent
+    to instantiate them with \textbf{unsuited types}, and the resulting
+    compilation errors are generally very long and hardly understandable.
+    As a last resort, the template authors are providing \textbf{documentation}
+    about the relevant parameters, and practice some tricky
+    \textbf{template meta-programmation}. C++20 finally brings simpler ways to
+    define constraints on template parameters !.
+  \end{block}
+\end{frame}
+
+\begin{frame}[fragile]
+  \frametitlecpp[20]{C++20 Requirements and Concepts}
+  \begin{block}{Requirements and concepts in a nutshell}
+    \begin{itemize}
+    \item A template can now define requirements on its parameters.
```

Comment:
Let's try to lighten a bit each line to have less text. Here is could become
```suggestion
    \item template parameters can have requirements
```

Reply:
Again, check the new version.